### PR TITLE
New site design as a front page

### PR DESCRIPTION
**Pull request by leouieda**
Redesign the website as a front page for the whole project, including links and information about all the new packages. Keep a link and some information about the old `fatiando` library and link tot the blog post explaining why we're moving away from it.

### CODE REVIEW

**Comment by leouieda** [comment]
@fatiando/contributors I finally got around to fixing our front page to reflect the recent re-alignment of the project. I'd love some feed back on this before I make it live. It's a single page with information and links to the new projects we're developing (verde, harmonica, pooch, etc). A preview is available at https://www.fatiando.org/website/

File: index.rst [hunk 1]
```diff
@@ -1,75 +1,323 @@
...
-   people/uieda/index.rst
+We maintain several Python libraries which are in various stages of development: from
+early design to polished and published products. We'll also be adding new projects to
+our toolkit in the future (see how you can :ref:`get involved <contribute>`).
+
+.. raw:: html
+
+   <div class="row">
+      <div class="col-md-6 project">
+         <img class="project-logo" src="_static/verde-logo.svg">
```
Comment (santisoler) [inline index.rst]: @leouieda Would be nice to add link to the projects' documentation into theirs images.

Something like:
```html
<a href="http://www.fatiando.org/verde">
    <img class="project-logo" src="_static/verde-logo.svg">
</a>
```

**Comment by santisoler** [comment]
By the way, nice work!! The single page improves navigation! :+1:

File: index.rst (hunk 1 above)
Comment (leouieda) [reply index.rst]: Right, I meant to do that but forgot. Thanks!

**Comment by leouieda** [comment]
fatiando-new-site

**Pull request merged**